Find a functionally equivalent code:android.view.accessibility.AccessibilityEvent.clear:COMMENT
Method Modifier: protected   hidden      
Comment:/**
 * Clears the state of this instance.
 *
 * @hide
 */

@@ -6,13 +6,15 @@
     mContentChangeTypes = 0;
     mPackageName = null;
     mEventTime = 0;
-    while (!mRecords.isEmpty()) {
-        AccessibilityRecord record = mRecords.remove(0);
-        record.recycle();
+    if (mRecords != null) {
+        while (!mRecords.isEmpty()) {
+            AccessibilityRecord record = mRecords.remove(0);
+            record.recycle();
+        }
     }
 }

